### ISSUE TITLE
Adding Context parameter to resolver causes errors in introspection

### ISSUE THREAD

**Issue by mdnorman** [pgutkowski/KGraphQL#47]
Adding a Context parameter to a resolver seems to work properly during execution, but introspection comes back with an object that fails to be parsed properly by eg GraphiQL.

Example resolver:
```
mutation("registerAccount") {
  accessRule { ctx ->
    println("rule context: $ctx")
    null
  }

  suspendResolver { ctx: Context, email: String ->
    println("resolver context: $ctx")
    println("email: $email")
  }
}
```

When executing the function, it does the right thing:
```
rule context: com.github.pgutkowski.kgraphql.Context@200146c2
resolver context: com.github.pgutkowski.kgraphql.Context@200146c2
email: some-person@example.com
```

However, GraphiQL gives the following error when introspecting the schema: `Error: Unknown type reference: {"kind":"OBJECT","name":null,"ofType":null}`

In addition, executing with "extra" parameters shows that `ctx` is incorrectly included in the parameter list: `registerAccount does support arguments [ctx, email]. Found arguments [accountName, email]

**Comment by mdnorman** [comment]
Created a PR: https://github.com/pgutkowski/KGraphQL/pull/48

**Comment by pgutkowski** [comment]
Merged #48, It will be included in upcoming 0.3.1 release, planned for today.

**Comment by mdnorman** [comment]
Awesome, thanks!
